lldb/include/lldb/Target/ThreadPlan.h

```diff
@@ -395,6 +396,11 @@ class ThreadPlan : public std::enable_shared_from_this<ThreadPlan>,
 
   bool IsControllingPlan() { return m_is_controlling_plan; }
 
+  // Returns true if this plan is a leaf plan, meaning the plan will be popped
```

Comment:
Leaf plans are only auto-popped if they don't explain the stop, right?  Might be worth mentioning that here.